topological_navigation/scripts/localisation.py

```diff
@@ -37,6 +37,10 @@ def __init__(self, name) :
         self.rec_map=False
         self.loc_by_topic=[]
         
+        self.current_pose=Pose()
+        self.previous_pose=Pose()
+        self.previous_pose.position.x=1000 #just give a random big value so this is tested 
```

Comment:
Yes, if not when the robot is initialised at 0,0 it doesn't  test localise by topic I don't think we will have maps bigger than 1 km will find a nicer solution afterwards